setup/native-script.ps1

```diff
@@ -34,6 +37,12 @@ if (!$env:JAVA_HOME) {
 
 # install NativeScript CLI
 write-host -BackgroundColor Black -ForegroundColor Yellow "Installing NativeScript CLI"
-npm install -g nativescript
+
+$oldPathUser = [Environment]::GetEnvironmentVariable("PATH", "User")
+$pathMachine = [Environment]::GetEnvironmentVariable("PATH", "Machine")
+$myPath = [Environment]::GetEnvironmentVariable("PATH")
+[Environment]::SetEnvironmentVariable("PATH", "$myPath;$oldPathUser;$pathMachine;$env:ProgramFiles\nodejs")
```

Comment:
I've not tested it, but according to [this forum](http://www.vistax64.com/powershell/239765-how-set-systems-environment-variable-permanently.html#post1101062) the PATH will not be "set" in the current Shell until you restart:
```
To set it PERMANENTLY you use SetEnvironmentVariable, but to have it
take effect NOW you need to use the Env: drive.
```